tests/fixtures/fhir-gdc-examples/META/Patient.ndjson

```diff
@@ -0,0 +1 @@
+{"resourceType":"Patient","id":"ccf801ae-7d28-5446-9c98-d79185ce2eb1","extension":[{"url":"http://hl7.org/fhir/us/core/StructureDefinition/us-core-birthsex","valueCode":"F"},{"extension":[{"url":"ombCategory","valueCoding":{"system":"urn:oid:2.16.840.1.113883.6.238","code":"2106-3","display":"White"}},{"url":"https://ncit.nci.nih.gov","valueCoding":{"system":"https://ncit.nci.nih.gov/ncitbrowser/ConceptReport.jsp?dictionary=NCI_Thesaurus&ns=ncit&code=C41261","code":"C41261","display":"white"}}],"url":"http://hl7.org/fhir/us/core/StructureDefinition/us-core-race","valueString":"White"},{"extension":[{"url":"ombCategory","valueCoding":{"system":"urn:oid:2.16.840.1.113883.6.238","code":"2186-5","display":"Not Hispanic or Latino"}},{"url":"https://ncit.nci.nih.gov","valueCoding":{"system":"https://ncit.nci.nih.gov","code":"C41222","display":"not hispanic or latino"}}],"url":"http://hl7.org/fhir/us/core/StructureDefinition/us-core-ethnicity","valueString":"not hispanic or latino"},{"url":"http://hl7.org/fhir/SearchParameter/patient-extensions-Patient-age","valueQuantity":{"value":72}}],"identifier":[{"system":"https://gdc.cancer.gov/case_submitter_id","value":"TCGA-38-7271"},{"system":"https://gdc.cancer.gov/case_id","value":"8214a0d1-5e2d-4a7a-acb1-e5580755db83"}],"gender":"female","birthDate":"1934-01-01","deceasedDateTime":"1970-01-01T00:33:28+00:00"}
```

Comment:
Noticed there's an extension nested in an extension, was this intentional?

Reply:
Legal FHIR:

Any resource can have an extension:
<img width="1138" alt="image" src="https://github.com/user-attachments/assets/89ea0a81-f685-4c77-b50e-fb0f7e981cd6">

An extension is an element, that in turn can have an extension:
<img width="902" alt="image" src="https://github.com/user-attachments/assets/7ab81ac6-dc02-45d8-9f21-331c0b883349">

Handled here:
https://github.com/ACED-IDP/gen3_util/blob/feature/stateful-emitter/tests/unit/dataframer/dataframer.py#L47-L48